x/programs/rust/examples/token/src/lib.rs

```diff
@@ -100,6 +100,18 @@ pub fn transfer(program: Program, sender: Address, recipient: Address, amount: i
     true
 }
 
+/// Mints tokens to multiple recipients.
+#[public]
+pub fn mint_to_many(program: Program, recipients: Vec<Address>, amounts: Vec<i32>) -> bool {
```

Comment:
Can we change this so that you can pass in `Vec<(Address, Amount)>`? And can you have that done by EOD, please? Haha

Jokes aside, do you think this would be possible? If yes, I'll create an issue!

Reply:
possible, on it!

Reply:
changed